pkg/rpc/mock.go

```diff
@@ -45,6 +57,16 @@ type CurrentChallenge struct {
 	Tip           *big.Int
 }
 
+// CurrentReserves holds details about the current reserves on the uniswap pair contract.
+// Note: Each Uniswap smart contract, or pair, manages a liquidity pool made up of reserves of two ERC-20 tokens.
```

Comment:
Reserve here is how much liquidity is in the pool? If yes please add this extra details.